### PR TITLE
Add noobaa sso test migration to cypress

### PR DESCRIPTION
**Pull request by TimothyAsirJeyasing**
Signed-off-by: Timothy Asir Jeyasingh <tjeyasin@redhat.com>

### CODE REVIEW

**Comment by TimothyAsirJeyasing** [comment]
@bipuladh , @afreen23 Please review

File: frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/noobaa-sso.spec.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { checkErrors } from '../../../integration-tests-cypress/support';
+
+describe('Check noobaa link in obejct service dashboard and perform SSO', () => {
+  before(() => {
```
Comment (bipuladh) [inline frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/noobaa-sso.spec.ts:4]:
```suggestion
  before(() => {
      cy.install();
```

**Review by bipuladh — APPROVED** [review]
One minor change.
Everything else looks good
/ok-to-test

File: frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/noobaa-sso.spec.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    checkErrors();
+  });
+
+  after(() => {
+    cy.logout();
+  });
+
+  it('Test admin user login for noobaa', () => {
+    //cy.visit('/dashboards/object-service');
+    cy.byLegacyTestID('system-name-mcg').eq(0)
```
Comment (bipuladh) [inline frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/noobaa-sso.spec.ts:19]: Let's update the UI to make sure `data-test-id` is not reused inside the same space.
```suggestion
    cy.byLegacyTestID('system-name-mcg')
```

**Comment by a2batic** [comment]
@TimothyAsirJeyasing please squash the commits.

File: frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/noobaa-sso.spec.ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import { checkErrors } from '../../../integration-tests-cypress/support';
+
+describe('Check noobaa link in obejct service dashboard and perform SSO', () => {
+  before(() => {
+      cy.install();
+    cy.login();
```
Comment (afreen23) [inline frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/noobaa-sso.spec.ts:6]: You should have login() first and then install()
```suggestion
    cy.login();
    cy.install();
```

File: frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/noobaa-sso.spec.ts [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+  afterEach(() => {
+    checkErrors();
+  });
+
+  after(() => {
+    cy.logout();
+  });
+
+  it('Test admin user login for noobaa', () => {
```
Comment (afreen23) [inline frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/noobaa-sso.spec.ts:18]: We dont want to modify existing test cases but migrate them to cypress.
https://github.com/openshift/console/blob/36558f21ee5161b36e6da6d19ef0da2adaf8cf28/frontend/packages/ceph-storage-plugin/integration-tests/tests/2-tests/noobaa-sso-scenario.ts#L1

**Review by afreen23 — CHANGES_REQUESTED** [review]

File: frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/noobaa-sso.spec.ts [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+describe('Check noobaa link in obejct service dashboard and perform SSO', () => {
+  before(() => {
+    cy.login();
+    cy.install();
+    cy.visit('/dashboards/object-service');
+  });
+
+  afterEach(() => {
+    checkErrors();
```
Comment (bipuladh) [inline frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/noobaa-sso.spec.ts:11]:
```suggestion
    cy.checkErrors();
```

File: frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/noobaa-sso.spec.ts [hunk 6]
```diff
@@ -0,0 +1,30 @@
+import { checkErrors } from '../../../integration-tests-cypress/support';
```
Comment (bipuladh) [inline frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/noobaa-sso.spec.ts:1]:
```suggestion
```

File: frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/noobaa-sso.spec.ts [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+  it('Check that noobaa dashboard is opening and links available.', () => {
+    cy.byLegacyTestID('system-name-mcg')
+      .invoke("attr", "href")
+      .then(href => {
+        cy.request(href)
+          .then((response) => {
+            expect(response.status).to.eq(200)
+            expect(response.body).to.contain(
+              'NooBaa Management Console')
+          });
```
Comment (bipuladh) [inline frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/noobaa-sso.spec.ts:27]:
```suggestion
            expect(response.status).toEqual(200)
            expect(response.body).contains(
              'NooBaa Management Console')
          });
```
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_console/7570/pull-ci-openshift-console-master-frontend/1340992172702306304#1:build-log.txt%3A1485

**Comment by bipuladh** [comment]
@TimothyAsirJeyasing please squash the commits. Rest of the PR looks good.
/retest

**Pull request merged**